db/schema.rb

```diff
@@ -74,6 +74,8 @@
     t.integer "number_of_word_files"
     t.boolean "dirty", default: false
     t.string "status", default: "live"
+    t.integer "number_of_spelling_errors"
+    t.integer "number_of_readability_issues"
```

Comment:
I think I will change these to just `spelling_count` `readability_count` etc. What do you think @pmanrubia?